<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QWidget" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Whisper - LogIn</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Icons/Resource/Icon/ic_LOGO_64.png</normaloff>:/Icons/Resource/Icon/ic_LOGO_64.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="log_in">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>379</y>
     <width>161</width>
     <height>37</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(255,255, 200,80);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="text">
    <string>LOG IN</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>701</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>137</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,140);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_enter">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>170</y>
     <width>161</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(0, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_enter">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>230</y>
     <width>161</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>86</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255,0);
color: rgb(168, 185, 196);</string>
   </property>
   <property name="text">
    <string>W H A V E T A L K</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(204, 224, 251, 50);
</string>
   </property>
   <property name="text">
    <string>Remember</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>76</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(32, 101, 144);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>170</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255, 255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Icons/Resource/Icon/ic_username.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>230</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255, 255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Icons/Resource/Icon/ic_password.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>358</y>
     <width>301</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 68px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/botton.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>312</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255, 255,0);
color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Icons/Resource/Icon/ic_register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_button">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>313</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);
color: rgb(86, 96, 109);</string>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>  Incorrect login/password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="errorconnect_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>416</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>Error connecting...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/header.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_window">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>3</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_close_brown.png</normaloff>:/Icons/Resource/Icon/ic_close_brown.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="turn_window">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_turn_brown.png</normaloff>:/Icons/Resource/Icon/ic_turn_brown.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255, 255,0);
color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Icons/Resource/Icon/ic_login.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>checkBox</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>username_enter</zorder>
  <zorder>pass_enter</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>label_7</zorder>
  <zorder>label_5</zorder>
  <zorder>label_8</zorder>
  <zorder>log_in</zorder>
  <zorder>label_2</zorder>
  <zorder>reg_button</zorder>
  <zorder>error_label</zorder>
  <zorder>errorconnect_label</zorder>
  <zorder>label_9</zorder>
  <zorder>close_window</zorder>
  <zorder>turn_window</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
